OSDOCS-10803: Added Proxy support section in oc-mirror v2 docs (#88672)
openshift/openshift-docs: modules/oc-mirror-proxy-support.adoc

// Module included in the following assemblies:
//
// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

:_mod-docs-content-type: CONCEPT
[id="oc-mirror-proxy-support_{context}"]
= oc-mirror plugin v2 support proxy setting

> The oc-mirror plugin v2 can operate in a proxy-configured environment, utilizing the system proxy settings to retrieve all images, including those for the {product-title} platform, Operator catalog, and `additionalImages`.

Reviewer: ```suggestion
The oc-mirror plugin v2 can operate in a proxy-configured environment. The plugin can use system proxy settings to retrieve all images, such as those images for {product-title} platform, Operator catalog, and the `additionalImages` registry.
```

Reviewer: IBM SG: 

![Screenshot From 2025-02-17 16-10-20](https://github.com/user-attachments/assets/5fa90ea8-0911-4fe8-bd95-e302f790814b)

// Module included in the following assemblies:
//
// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

:_mod-docs-content-type: CONCEPT
[id="oc-mirror-proxy-support_{context}"]
= oc-mirror plugin v2 support proxy setting

> The oc-mirror plugin v2 can operate in a proxy-configured environment. The plugin can use system proxy settings to retrieve all images, such as those images for {product-title} platform, Operator catalog, and the `additionalImages` registry.

Reviewer 2: ```suggestion
The oc-mirror plugin v2 can operate in a proxy-configured environment. The plugin can use system proxy settings to retrieve images for {product-title}, Operator catalog, and the `additionalImages` registry.
```

- "{product-title} platform" is rendering as: "OpenShift Container Platform platform".

Author: Done.
